commands/resource_commands/deploy

```diff
@@ -0,0 +1,247 @@
+        status_log $TYPE_DEFAULT  "relation_name: ${relation_name}"
+
+        local register_inclusion_resolver=$(gum choose --header "Allow Customers queries to include data from related Employees instances?" "Yes" "No") || exit $?
+        status_log $TYPE_DEFAULT  "register_inclusion_resolver: ${register_inclusion_resolver}"
+        
+        if [ "$register_inclusion_resolver" == "Yes" ]; then
+            register_inclusion_resolver=true
+        else
+            register_inclusion_resolver=false
+        fi
+
+        RELATIONS="$RELATIONS
+  - name: \"$relation_name\"
+    spec:
+      relationName: \"$relation_name\"
+      sourceModel: \"$source_model\"
+      destinationModel: \"$target_model\"
+      foreignKeyName: \"$foreign_key\"
+      relationType: \"$relation_type\"
+      registerInclusionResolver: $register_inclusion_resolver
+      "
+
+        status_log $TYPE_INFO "Do you want to add another rule?"
+        local another_rule=$(gum choose "Yes" "No") || exit $?
+        
+        if [ "$another_rule" == "No" ]; then
+            return 0
+        fi
+
+    done
+    
+}
+
+deploy_db_file_template() {
+
+    if [ "$(kubectl config current-context)" = "minikube" ]; then
+        echo "patching for minikube..."
+        MINIKUBE_PATCH=" --set gras.ssl=false --set grapi.ssl=false --set gruim.ssl=false "
+    else
+        MINIKUBE_PATCH=""
+    fi
+
+    echo "check xrds are available"
+    status_log $TYPE_INFO "wait for grapi to be deployed"
+    CRD=grapi && echo "wait for $CRD to be deployed:" && until kubectl explain $CRD >/dev/null 2>&1; do echo -n .; sleep 1; done && echo "$CRD deployed"
+    status_log $TYPE_SUCCESS "grapi is deployed"
+    status_log $TYPE_INFO "wait for compositegrappleapis to be deployed"
+    CRD=compositegrappleapis && echo "wait for $CRD to be deployed:" && until kubectl explain $CRD >/dev/null 2>&1; do echo -n .; sleep 1; done && echo "$CRD deployed"
+    status_log $TYPE_SUCCESS "compositegrappleapis is deployed"
+    status_log $TYPE_INFO "wait for composition/grapi.grsf.grpl.io to be deployed"
+    CRD=composition/grapi.grsf.grpl.io && echo "wait for $CRD to be deployed:" && until kubectl get $CRD >/dev/null 2>&1; do echo -n .; sleep 1; done && echo "$CRD deployed"
+    status_log $TYPE_SUCCESS "composition/grapi.grsf.grpl.io is deployed"
+    status_log $TYPE_INFO "wait for composition/muim.grsf.grpl.io to be deployed"
+    CRD=composition/muim.grsf.grpl.io && echo "wait for $CRD to be deployed:" && until kubectl get $CRD >/dev/null 2>&1; do echo -n .; sleep 1; done && echo "$CRD deployed"
+    status_log $TYPE_SUCCESS "composition/muim.grsf.grpl.io is deployed"
+
+    gum spin --title "Deploying db-file-template" -- helm upgrade --install ${DB_FILE_NS} oci://public.ecr.aws/${awsregistry}/gras-deploy -n ${DB_FILE_NS} -f /tmp/db-file.yaml ${MINIKUBE_PATCH} --create-namespace 
```

Comment:
this installes the template over the example deployment installed with "grpl e d".
what we want is to create a new deployment in a new namespace.
the new deployment should have the name of the resource specified an shall be installed into the current namespace:
so:
`gum spin --title "Deploying grapple resource" -- helm upgrade --install ${GRAS_NAME} oci://public.ecr.aws/${awsregistry}/gras-deploy  -f /tmp/db-file.yaml ${MINIKUBE_PATCH}`